src/raven.js

```diff
@@ -413,6 +413,16 @@ Raven.prototype = {
             timestamp: now() / 1000
         }, obj);
 
+        if (isFunction(this._globalOptions.breadcrumbCallback)) {
+            var result = this._globalOptions.breadcrumbCallback(crumb);
+
+            if (result) {
+                crumb = result;
+            } else {
```

Comment:
To be consistent with dataCallback, I'm also using the original value if the callback returns an empty object.